pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -663,6 +663,11 @@ const (
 	DNSRecordExternalName = "external"
 	// DNSRecordOwnerName is a constant for DNSRecord objects used for the owner domain name.
 	DNSRecordOwnerName = "owner"
+
+	// Amd64 is a constant for the 'amd64' architecture.
+	Amd64 = "amd64"
+	// Arm64 is a constant for the 'arm64' architecture.
+	Arm64 = "arm64"
```

Comment:
This is a common change with https://github.com/gardener/gardener/pull/6178.
Which one do you want to get merged first?
Please mark the other one as draft accordingly.